1.0.1/jessie/build/README.md

```diff
@@ -19,4 +19,59 @@ This image contains:
 
 ## Supported tags
 
-- [`1.0.1`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0.1/jessie/build/Dockerfile)
\ No newline at end of file
+- [`1.0.1`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0.1/jessie/build/Dockerfile)
+
+## Example Usage
+
+1. Create a Dockerfile to build your application (`Dockerfile.build` is a common name used).
+
+    ```Dockerfile
+    FROM microsoft/aspnetcore-build
+    WORKDIR /app
+
+    COPY project.json .
+    RUN dotnet restore
+
+    COPY . .
+    RUN dotnet publish --output /out/. --configuration Release
+    ```
+
+2. Build your image:
+
+    ```
+    $ docker build -t build-image -f Dockerfile.build .
+    ```
+
+3. Create a container from your image and copy your built application out.
+
+    ```
+    $ docker create --build-cont build-image
+    $ docker cp build-image:/out ./output
+    ```
+
+Now you have built your ASP.NET Core application inside a container and have the published output on the host ready to deploy. From here you could then construct an optimized runtime image with the `microsoft/aspnetcore` image or just deploy/run the binaries as normal without using Docker at runtime.
+
+###TODO: I think everything below this line we could cut, and maybe have them as an example in a samples repo. In fact I think a set of samples that show using this technique end-to-end should be published.
```

Comment:
Richard sent me a link to that, I am about to send a PR to it to start a sample that does what I am describing here with a console app and the dotnet images.